config/kubernetes/sig-release/teams.yaml

```diff
@@ -178,3 +178,13 @@ teams:
         - justaugustus
         - tpepper
         privacy: closed
+  publishing-bot-reviewers:
+    description: Reviews for publishing-bot
+    maintainers:
+    - caesarxuchao
+    - nikhita
```

Comment:
maintainers = approvers
members = reviewers

I've added myself here because the GH API requires that org owners be in the maintainers list. They can't be added as members. :/

Reply:
While we're converting maintainers -> members (https://github.com/kubernetes/org/issues/563), it may be good to add `caesarxuchao` to the members right away instead of the maintainers section 😄 (org owners aside)

Reply:
Thanks @rlenferink. I've updated as per https://github.com/kubernetes/org/issues/563.